### DISCUSSION TITLE
How does line zone get triggered?

### DISCUSSION THREAD

**Discussion by abichoi** [roboflow/supervision#1452] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Hi, I am trying to under how the line zone get triggered. 
Let's say I set the triggering_anchors as BOTTOM_LEFT.  Does xy of the bottom_left corner of the detection box of the object has to cross the line exactly to trigger the line zone?  For example, does the bottom left corner needs to be one pixel below the line in frame 1,  then the corner is on the line in frame 2, and then the corner is one pixel above the line in frame 3, to trigger the line?

Or is there a buffer zone? For example, if the bottom_left corner is a few pixels BELOW the line in frame 1 and it is a few pixels ABOVE the line in frame 2, will it trigger the line?

Sorry I am aware how confusing my question sounds...

**Comment by LinasKo** [comment]
Hi @abichoi 👋

I'm sorry for not getting back to you sooner. That is a valid question!

Given a set of anchors, ALL anchors of an object need to cross the line for it to count as a cross.
In terms of precision, there is a 2px strip around the line (1 px on each side), but I have yet to test how well it works when dealing with objects very close to the line.
